R/board_rsconnect.R

```diff
@@ -388,17 +388,20 @@ board_pin_find.pins_board_rsconnect <- function(board,
 
 # Content -----------------------------------------------------------------
 
+rsconnect_content_cache_env <- rlang::new_environment()
+rsconnect_user_cache_env <- rlang::new_environment()
```

Comment:
I put these here? And also I think it's simplest to still have two of them, one for the pieces of content and one for the users.

Reply:
I think this is a totally reasonable approach but we're starting to build a bit of a consensus around using [`the`](https://github.com/tidyverse/design/issues/126) as a sentinel for a mutate package environment. If you wanted to use that, I think you'd do:

```R
the <- rlang::new_environment()
the$content_cache <- rlang::new_environment()
the$user_cache <-  rlang::new_environment()
```